Missionframework/modules/15_build/fnc/fn_build_loadData.sqf

```diff
@@ -21,28 +21,18 @@ if (KPLIB_param_debug) then {diag_log "[KP LIBERATION] [SAVE] Build module loadi
 
 private _moduleData = ["build"] call KPLIB_fnc_init_getSaveData;
 
-// Check if there is a new campaign
-if (_moduleData isEqualTo []) then {
-    if (KPLIB_param_debug) then {diag_log "[KP LIBERATION] [SAVE] Build module data empty, creating new data...";};
+// DEPRECATED, persistence is handled by dedicated module
+// TODO remove later. For now this function is left for save compatiblity.
```

Comment:
If it's not vital for your testings, we shouldn't implement "save compatibility" stuff between sprint pre-releases.